docs/administrators/on_disk_layout.rst

```diff
@@ -0,0 +1,70 @@
+On Disk Layout
+====================
+
+The server stores
+ - all data below the state directory, as configured by :inmanta.config:option:`config.state-dir`,
+which defaults to `/var/lib/inmanta`
+ - all logs below the log directory, as configured by  :inmanta.config:option:`config.log-dir`,
+which defaults to `/var/log/inmanta`
+
+
+.. code-block::
+
+    /var/lib/inmanta
+       ├─ server/
+           ├─ .inmanta_use_new_disk_layout   # marker file for new disk layout
+           ├─ env_uuid_1/                    # uuid for this environment
+           │   ├─ executors/                 # storage for all executors
+           │   │   ├─ venvs/                 # python virtual envs for all executors
+           │   │   │  ├─ venv_blueprint_hash_1/
+           │   │   │  ├─ venv_blueprint_hash_2/
+           │   │   │  ├─ ...
+           │   │   ├─ code/                  # Handler code for all executors
+           │   │      ├─ executor_blueprint_hash_1/
+           │   │      ├─ executor_blueprint_hash_2/
+           │   │      ├─ ...
+           │   ├─ scheduler.cfg              # Configuration for the scheduler for this environment
+           │   ├─ compiler/                  # Folder for the compiler, containing a checkout of the project for this environment
+           │   │  ├─ .env                    # symlink to .env-py3.12, updated by the server when required
+           │   │  ├─ .env-py3.12             # Virtual env for the compiler to use, with python version
+           │
+           ├─ env_uuid_2/
+           │   ├─ ( ... )
+    /var/log/inmanta
+       ├─ server.log                         # server main log file
+       ├─ resource-actions-env_uuid_1.log    # deploy audit log for environment env_uuid_1, also available via API
+       ├─ callback.log                       # log of all http callbacks performed by lsm
```

Comment:
This is an LSM only log file. Like this we will put in into the OSS docs as well. Should we main two tables?

Reply:
I can live with having one file too many ;-)